### PR TITLE
New tools, books and blogs

### PR DESCRIPTION
**Pull request by jarv-git**
Added Shikra under Hardware Tools. Created new Bluetooth/BLE, ZigBee, SDR, and RFID/NFC tools subsections. Added 5 books. Added 6 blogs.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -61,78 +74,140 @@ Software tools for analyzing embedded/IoT firmware.
...
 - [J-Link](https://www.segger.com/products/debug-probes/j-link/models/model-overview/) - J-Link offers USB powered JTAG debug probes for multiple different CPU cores :euro:.
 
+## Bluetooth BLE Tools
+- [UberTooth One](https://greatscottgadgets.com/ubertoothone/) - Open source 2.4 GHz wireless development platform suitable for Bluetooth experimentation.
+- [Bluefruit LE Sniffer](https://www.adafruit.com/product/2269) - Easy to use Bluetooth Low Energy sniffer.
+
+## ZigBee Tools
+
+- [ApiMote](http://apimote.com) - ZigBee security research hardware for learning about and evaluating the security of IEEE 802.15.4/ZigBee systems. Killerbee compatible.
+- [Atmel RZUSBstick] - Discontinued product. Lucky if you have one! - Tool for development, debugging and demonstration of a wide range of low power wireless applications including IEEE 802.15.4, 6LoWPAN, and ZigBee networks. Killerbee compatible.
```
Comment (0xricksanchez) [inline readme.md:101]: Remove square brackets or fix link

File: readme.md [hunk 2]
```diff
@@ -61,78 +74,140 @@ Software tools for analyzing embedded/IoT firmware.
...
+- [Freakduino](https://freaklabsstore.com/index.php?main_page=product_info&cPath=22&products_id=219&zenid=fpmu2kuuk4abjf6aurt3bjnfk4) - Low Cost Battery Operated Wireless Arduino Board that can be turned into a IEEE 802.15.4 protocol sniffer.
+
+## SDR Tools
+- [HackRF One](https://greatscottgadgets.com/harkrf/) - Software Defined Radio peripheral capable of transmission or reception of radio signals from 1 MHz to 6 GHz (half-duplex).
```
Comment (0xricksanchez) [inline readme.md:105]: link is dead?

File: readme.md [hunk 3]
```diff
@@ -61,78 +74,140 @@ Software tools for analyzing embedded/IoT firmware.
...
+- [YardStick One](https://greatscottgadgets.com/yardstickone/) - Half-duplex sub-1 GHz wireless transceiver.
+- [LimeSDR](https://www.crowdsupply.com/lime-micro/limesdr) - Software Defined Radio peripheral capable of transmission or reception of radio signals from 100 KHz to 3.8 GHz (full-duplex).
+- [BladeRF 2.0](https://www.nuand.com/bladerf-2-0-micro/) - Software Defined Radio peripheral capable of transmission or reception of radio signals from 47 MHz to 6 GHz (full-duplex).
+- [USRP B Series](https://www.ettus.com/product-categories/usrp-bus-series/) - Software Defined Radio peripheral capable of transmission or reception of radio signals from 70 MHz to 6 GHz (full-duplex).
+
+##RFID NFC Tools
+- [Proxmark 3 RDV4](https://www.proxmark.com/) - Powerful general purpose RFID tool. From Low Frequency (125kHz) to High Frequency (13.56MHz) tags
```
Comment (0xricksanchez) [inline readme.md:112]: Descriptions of list items have to end with a '.'

File: readme.md [hunk 4]
```diff
@@ -61,78 +74,140 @@ Software tools for analyzing embedded/IoT firmware.
...
+- [Freakduino](https://freaklabsstore.com/index.php?main_page=product_info&cPath=22&products_id=219&zenid=fpmu2kuuk4abjf6aurt3bjnfk4) - Low Cost Battery Operated Wireless Arduino Board that can be turned into a IEEE 802.15.4 protocol sniffer.
+
+## SDR Tools
+- [HackRF One](https://greatscottgadgets.com/harkrf/) - Software Defined Radio peripheral capable of transmission or reception of radio signals from 1 MHz to 6 GHz (half-duplex).
+- [YardStick One](https://greatscottgadgets.com/yardstickone/) - Half-duplex sub-1 GHz wireless transceiver.
+- [LimeSDR](https://www.crowdsupply.com/lime-micro/limesdr) - Software Defined Radio peripheral capable of transmission or reception of radio signals from 100 KHz to 3.8 GHz (full-duplex).
+- [BladeRF 2.0](https://www.nuand.com/bladerf-2-0-micro/) - Software Defined Radio peripheral capable of transmission or reception of radio signals from 47 MHz to 6 GHz (full-duplex).
+- [USRP B Series](https://www.ettus.com/product-categories/usrp-bus-series/) - Software Defined Radio peripheral capable of transmission or reception of radio signals from 70 MHz to 6 GHz (full-duplex).
+
+##RFID NFC Tools
```
Comment (0xricksanchez) [inline readme.md:111]: Broken header

File: readme.md [hunk 5]
```diff
@@ -61,78 +74,140 @@ Software tools for analyzing embedded/IoT firmware.
...
+- [Proxmark 3 RDV4](https://www.proxmark.com/) - Powerful general purpose RFID tool. From Low Frequency (125kHz) to High Frequency (13.56MHz) tags
+- [ChamaleonMini](http://chameleontiny.com/) - Programmable, portable tool for NFC security analysis.
+- [HydraNFC](https://hydrabus.com/hydranfc-1-0-specifications/) - Powerful 13.56MHz RFID / NFC platform. Read / write / crack / sniff / emulate.
 
 ## Books
 
-- 2020, Jasper van Woudenberg, Colin O'Flynn: [The Hardware Hacking Handbook: Breaking Embedded Security with Hardware Attacks](https://www.amazon.com/Hardware-Hacking-Handbook-Breaking-Embedded-ebook/dp/B077WZBFYL)
-- 2019, Yago Hansen: [The Hacker's Hardware Toolkit: The best collection of hardware gadgets for Red Team hackers, Pentesters and security researchers](https://www.amazon.com/Hackers-Hardware-Toolkit-collection-researchers/dp/1099209463)
+- 2020, Fotios Chantzis,  Evangel Deirme, Ioannis Stais, Paulino Calderon, Beau Woods: [Practical IoT Hacking](https://www.amazon.com/Fotios-Chantzis-ebook/dp/B085BVVSN6/)
```
Comment (0xricksanchez) [inline readme.md:118]: Try to avoid links from amazon and alternatively use the publishers store for example

File: readme.md [hunk 6]
```diff
@@ -21,29 +21,42 @@ They are easy to use and you do not need to be an expert to get first meaningful
   - [Analysis Tools](#analysis-tools)
   - [Extraction Tools](#extraction-tools)
   - [Support Tools](#support-tools)
+  - [Misc Tools](#misc-tools)
 - [Hardware Tools](#hardware-tools)
+- [Bluetooth BLE Tools](#bluetooth-ble-tools)
+- [ZigBee Tools](#zigbee-tools)
+- [SDR Tools](#sdr-tools)
+- [RFID NFC Tools](#rfid-nfc-tools)
```
Comment (0xricksanchez) [inline readme.md:29]: Broken ToC item due to broken header formatting

File: readme.md [hunk 7]
```diff
@@ -61,78 +74,140 @@ Software tools for analyzing embedded/IoT firmware.
...
+- 2020, Jasper van Woudenberg, Colin O'Flynn: [The Hardware Hacking Handbook: Breaking Embedded Security with Hardware Attacks](https://nostarch.com/hardwarehacking)
+- 2019, Yago Hansen: [The Hacker's Hardware Toolkit: The best collection of hardware gadgets for Red Team hackers, Pentesters and security researchers](https://github.com/yadox666/The-Hackers-Hardware-Toolkit/blob/master/TheHackersHardwareToolkit.pdf)
 - 2019, Aditya Gupta: [The IoT Hacker's Handbook: A Practical Guide to Hacking the Internet of Things](https://www.apress.com/us/book/9781484242995)
 - 2018, Mark Swarup Tehranipoor: [Hardware Security: A Hands-on Learning Approach](https://www.elsevier.com/books/hardware-security/bhunia/978-0-12-812477-2)
-- 2017, Aditya Gupta, Aaron Guzman: [IoT Penetration Testing Cookbook](https://www.packtpub.com/networking-and-servers/iot-penetration-testing-cookbook)  
+- 2018, Mark Carney: [Pentesting Hardware - A Practical Handbook (DRAFT)](https://github.com/unprovable/PentestHardware)
+- 2018,  Qing Yang, Lin Huang [Inside Radio: An Attack and Defense Guide](https://www.amazon.com/Inside-Radio-Attack-Defense-Guide-ebook/dp/B07BKVKL87/)
```
Comment (0xricksanchez) [inline readme.md:124]: see earlier.

File: readme.md [hunk 8]
```diff
@@ -61,78 +74,140 @@ Software tools for analyzing embedded/IoT firmware.
...
+- 2017, Aditya Gupta, Aaron Guzman: [IoT Penetration Testing Cookbook](https://www.packtpub.com/networking-and-servers/iot-penetration-testing-cookbook)
 - 2017, Andrew Huang: [The Hardware Hacker: Adventures in Making and Breaking Hardware](https://nostarch.com/hardwarehackerpaperback)
 - 2016, Craig Smith: [The Car Hacker's Handbook: A Guide for the Penetration Tester](https://nostarch.com/carhacking)
+- 2015, Keng Tiong Ng: [The Art of PCB Reverse Engineering](https://www.amazon.com/Art-PCB-Reverse-Engineering-Standard/dp/151880053X/)
```
Comment (0xricksanchez) [inline readme.md:128]: see earlier.

File: readme.md [hunk 9]
```diff
@@ -61,78 +74,140 @@ Software tools for analyzing embedded/IoT firmware.
...
 - 2015, Nitesh Dhanjan: [Abusing the Internet of Things: Blackouts, Freakouts, and Stakeouts](https://shop.oreilly.com/product/0636920033547.do)
+- 2015, Joshua Wright , Johnny Cache: [Hacking Wireless Exposed](https://www.amazon.com/Joshua-Wright-ebook/dp/B00U1Y17OI/)
```
Comment (0xricksanchez) [inline readme.md:130]: see earlier.

File: readme.md (hunk 8 above)
Comment (jarv-git) [reply readme.md:128]: It's only available at Amazon. Anyway, I will change it to the writer's web-page.

File: readme.md (hunk 5 above)
Comment (jarv-git) [reply readme.md:118]: I know, however, the book has not been yet announce at the publishers webpage.

File: readme.md (hunk 1 above)
Comment (jarv-git) [reply readme.md]: Fix on the latest commit.

File: readme.md (hunk 2 above)
Comment (jarv-git) [reply readme.md]: Fix on the latest commit. It's was a typo (harkrf --> hackrf)

File: readme.md (hunk 3 above)
Comment (jarv-git) [reply readme.md]: Fix on the latest commit.

File: readme.md (hunk 4 above)
Comment (jarv-git) [reply readme.md]: Fix on the latest commit.

File: readme.md (hunk 6 above)
Comment (jarv-git) [reply readme.md:29]: Fix on the latest commit.

File: readme.md (hunk 7 above)
Comment (jarv-git) [reply readme.md]: Fix on the latest commit.

File: readme.md (hunk 8 above)
Comment (jarv-git) [reply readme.md]: Fix on the latest commit.

File: readme.md (hunk 9 above)
Comment (jarv-git) [reply readme.md]: Fix on the latest commit.

**Comment by 0xricksanchez** [comment]
Still seems to be a merge conflict in the Books section @jarv-git . Otherwise, LGTM now and linter does not complain either. What do you think @weidenba

**Comment by weidenba** [comment]
I resolved the merge conflict and converted the technology based tools to subsections. I think we can merge it now. The linter complains about `apimote.com` link being dead. However, I checked it manually and it seems to work.

**Comment by 0xricksanchez** [comment]
Merged and thanks for the contribution @jarv-git !

**Pull request merged**